### DISCUSSION TITLE
Incentivize Voting

### DISCUSSION THREAD

**Discussion by thenic95** [ADADAO/Documents#21] [Ideas]
Once our government script is live and we are able to vote on proposals how can we incentivize voting and encourage voter participation?

**Comment by TCTwastaken** [comment]
Maybe rewards can be offered when tokens are "locked" in the voting/proposal contract.

**Reply by thenic95** [reply]
I like the idea.
Would each voter just receive a flat bonus or how would you approach that?

I can see that working well with conviction voting. Might be good to implement in a future version of our governance scripts.
